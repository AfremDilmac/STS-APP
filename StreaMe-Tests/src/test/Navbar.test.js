import Navbar from "../components/Navbar";
import { shallow } from "enzyme";

test("should take snapshot of  Navbar", () => {
  const wrapper = shallow(<Navbar />);
  expect(wrapper).toMatchSnapshot();
});
