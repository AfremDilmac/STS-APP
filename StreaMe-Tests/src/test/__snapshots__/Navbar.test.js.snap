// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take snapshot of  Navbar 1`] = `
<nav
  className="w-full flex py-6 justify-between items-center navbar"
>
  <Link
    to="/"
  >
    <h1
      className="text-4xl font-extrabold dark:text-white"
    >
      StreaME
    </h1>
  </Link>
  <ul
    className="list-none sm:flex hidden justify-end items-center flex-1"
  >
    <li
      className="font-poppins font-normal cursor-pointer text-[16px] text-white mr-10"
      key="home"
      onClick={[Function]}
    >
      <a
        href="#home"
      >
        Home
      </a>
    </li>
    <li
      className="font-poppins font-normal cursor-pointer text-[16px] text-dimWhite mr-10"
      key="features"
      onClick={[Function]}
    >
      <a
        href="#features"
      >
        Features
      </a>
    </li>
    <li
      className="font-poppins font-normal cursor-pointer text-[16px] text-dimWhite mr-0"
      key="product"
      onClick={[Function]}
    >
      <a
        href="#product"
      >
        Product
      </a>
    </li>
  </ul>
  <div
    className="sm:hidden flex flex-1 justify-end items-center"
  >
    <img
      alt="menu"
      className="w-[28px] h-[28px] object-contain"
      onClick={[Function]}
      src="menu.svg"
    />
    <div
      className="hidden p-6 bg-black-gradient absolute top-20 right-0 mx-4 my-2 min-w-[140px] rounded-xl sidebar"
    >
      <ul
        className="list-none flex justify-end items-start flex-1 flex-col"
      >
        <li
          className="font-poppins font-medium cursor-pointer text-[16px] text-white mb-4"
          key="home"
          onClick={[Function]}
        >
          <a
            href="#home"
          >
            Home
          </a>
        </li>
        <li
          className="font-poppins font-medium cursor-pointer text-[16px] text-dimWhite mb-4"
          key="features"
          onClick={[Function]}
        >
          <a
            href="#features"
          >
            Features
          </a>
        </li>
        <li
          className="font-poppins font-medium cursor-pointer text-[16px] text-dimWhite mb-0"
          key="product"
          onClick={[Function]}
        >
          <a
            href="#product"
          >
            Product
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
